README.rst

```diff
@@ -169,6 +169,23 @@ Events
     # Get latest stream position
     client.events().get_latest_stream_position()
 
+Metadata
+~~~~~~~~
+
+.. code-block:: python
+
+    # Get metadata
+    client.file(file_id='SOME_FILE_ID').metadata().get()
+
+    # Create metadata
+    client.file(file_id='SOME_FILE_ID').metadata().create({'key', 'value')
+
+    # Update metadata
+    from boxsdk.object.metadata import MetadataUpdate
```

Comment:
Yeah, it shouldn't be so hard to do an update. Either from boxsdk import MetadataUpdate, or, perhaps better, add a method to Metadata that will give you a MetadataUpdate object.